Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerView.swift

```diff
@@ -42,34 +42,49 @@ open class _ChatMessageComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProv
         .messageComposer
         .documentAttachmentsCollectionView.init()
         .withoutAutoresizingMaskConstraints
-    
+
+    /// A view to input content of a message.
     public private(set) lazy var messageInputView = components
         .messageInputView.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var sendButton = components
+
+    /// A button to send a message.
+    public private(set) lazy var sendButton: UIButton = components
         .sendButton.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var confirmButton: UIButton = UIButton()
+
+    /// A button to confirm when editing a message.
+    public private(set) lazy var confirmButton: UIButton = components
+        .confirmButton.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var attachmentButton: UIButton = UIButton()
+
+    /// A button to open the user attachments.
+    public private(set) lazy var attachmentButton: UIButton = components
+        .attachmentButton.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var commandsButton: UIButton = UIButton()
+
+    /// A button to open the available commands.
+    public private(set) lazy var commandButton: UIButton = components
+        .commandsButton.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var shrinkInputButton: UIButton = UIButton()
+
+    /// A button to show or hide the left action buttons.
+    public private(set) lazy var shrinkInputButton: UIButton = components
+        .shrinkInputButton.init()
         .withoutAutoresizingMaskConstraints
-    
-    public private(set) lazy var dismissButton: UIButton = UIButton()
+
+    /// A button to dismiss the current state (quoting, editing, etc..).
+    public private(set) lazy var dismissButton: UIButton = components
+        .closeButton.init()
         .withoutAutoresizingMaskConstraints
-    
+
+    /// A label part of the header view to display the current state (quoting, editing, etc..).
     public private(set) lazy var titleLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withBidirectionalLanguagesSupport
```

Comment:
`.withAdjustingFontForContentSizeCategory`

Reply:
Fixed it here: https://github.com/GetStream/stream-chat-swift/pull/1048/commits/acbd51a33a87f6f5ca3f21bcdf4117542c1f0de3 ✅